smolyay/surrogate.py

```diff
@@ -0,0 +1,469 @@
+import numpy
+
+
+class SmolyakGrid:
```

Comment:
Shouldn't this class be removed now that there's a generator in the `grid` module?

Similarly, is there anything in `smolyak.py` that hasn't been moved somewhere else already? I think `smolyak.py` can now be removed too.